tests/test_coor_trans.py

```diff
@@ -77,3 +78,87 @@ def testRegisterRots(self):
         q_mat_est, flag_est = register_rotations(rots_ref, regrots_ref)
 
         self.assertTrue(np.allclose(flag_est, flag) and np.allclose(q_mat_est, q_mat))
+
+    def testSquareCrop2D(self):
```

Comment:
Maybe test a rectangular M x N matrix, M != N, with size in between M and N. That way you'll get coverage for the `else:` condition in `crop_2d` (line 334).